packages/optimistic-oracle/src/proposer.js

```diff
@@ -160,7 +160,7 @@ class OptimisticOracleProposer {
       priceRequest.ancillaryData,
       proposalPrice
     );
-    this.logger.error({
+    this.logger.debug({
```

Comment:
We can switch this log about "Detecting a price request" to debug now that the monitor emits an ERROR level log when a price request is detected.

Reply:
Would this get switched back once, presumably some point in the future, price requests no longer trigger an ERROR level log?

Reply:
yeah, i think its more stable long term as DEBUG as we get comfortable using the OO but now that the Monitor is tracking this same event we can deprecate ERROR > DEBUG